control-plane/outputs.tf

```diff
@@ -40,8 +40,8 @@ output "rancher_url" {
 }
 
 output "rancher_token" {
-  value     = module.k3s.rancher_token
-  sensitive = true
+  value     = var.sensitive_token ? null : nonsensitive(module.k3s.rancher_token)
+  sensitive = false
```

Comment:
Why has this changed?

Reply:
Being able to mark this token as sensitive/nonsensitive allows for it to be displayed in the terraform output which is useful when using the `clusters` module to provision downstreams. It speeds up the manual process of getting an api token and allows for some shell-based automation.